web/src/main/angular/src/app/core/components/transaction-timeline-v2/transaction-timeline-v2.component.ts

```diff
@@ -0,0 +1,116 @@
+import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+    selector: 'pp-transaction-timeline-v2',
+    templateUrl: './transaction-timeline-v2.component.html',
+    styleUrls: ['./transaction-timeline-v2.component.css']
+})
+
+export class TransactionTimelineComponentV2 implements OnInit {
+    @Input() keyIndex: { [key: string]: number};
+    @Input() startTime: number;
+    @Input() endTime: number;
+    @Input() barRatio: number;
+    @Output() outSelectTransaction: EventEmitter<string> = new EventEmitter();
+    @Input() rowData: any[];
```

Comment:
Change order and syntax
```suggestion
    @Input() rowData: any[];
    @Output() outSelectTransaction = new EventEmitter<string>();
```